lib/network/struct.go

```diff
@@ -18,7 +18,7 @@ import (
 
 // How many times should we try to connect
 const maxRetry = 10
-const waitRetry = 1 * time.Second
+const waitRetry = 100 * time.Millisecond
 const timeOut = 5 * time.Second
```

Comment:
Is this on purpose? Before we waited 1 sec for retrying and now 100ms (makes sense to me because one second is rather long but I am not sure)